### ISSUE TITLE
About experiment results

### ISSUE THREAD

**Issue by zhanqan** [Nanne/pytorch-NetVlad#7]
I have some questions about the experiment results you got, e.g. pytorch-NetVlad(vgg16) | 85.2 | 94.8 | 97.0, this was trained on Pittsburgh 250k or Pittsburgh 30k? I run main.py with train mode and default settings on  Pittsburgh 30k, but my test result is 82.29,91.46, 93.66.

**Comment by Nanne** [comment]
It's trained on the pittsburgh 30k train set, and evaluated on the 30k validation set. 

There is quite some randomness in the process, so could be that you get slightly different results. While 3-4% is more than slightly, I don't see any major reason for concern here. Experiment with training for longer, or see if you're overfitting etc.
